userguide/hugo.yaml

```diff
@@ -33,10 +33,10 @@ services:
 
 languages:
   en:
-    title: Docsy
```

Comment:
FYI, `title` isn't a custom parameter and shouldn't have been moved to be under `params`. In fact, we don't need a `title` parameter here, so it is being dropped in #1994.